src/components/theme/default/index.js

```diff
@@ -0,0 +1,15 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+import { palette } from "@cyverse-de/ui-lib";
+
+const theme = createMuiTheme({
+    palette: {
+        primary: {
+            main: palette.blue,
+        },
+        secondary: {
+            main: palette.lightBlue,
+        },
+    },
+});
+
+export default theme;
```

Comment:
We might want to define overrides all the intentions defined here - https://material-ui.com/customization/palette/

Reply:
This seems very useful - https://material-ui.com/customization/color/#color-tool

Reply:
Agreed on both counts. In the UX meeting yesterday, Mariah mentioned getting us access to the updated set of official CyVerse color values. I was planning on one of us going through and adding all of them to the theme once we have them.